src/go/configmanager/config_manager.go

```diff
@@ -45,24 +46,22 @@ var (
 					GCP metadata server will not be called to fetch access token, and
 					following flags will be ignored; --service_config_id, --service,
 					--rollout_strategy`)
-
-	// secured HTTP client calling service management service.
-	serviceConfigFetcherClient *http.Client
 )
 
 // Config Manager handles service configuration fetching and updating.
 // TODO(jilinxia): handles multi service name.
 type ConfigManager struct {
-	serviceName        string
-	serviceInfo        *configinfo.ServiceInfo
-	envoyConfigOptions options.ConfigGeneratorOptions
-	curRolloutID       string
-	curConfigID        string
+	serviceName                   string
+	serviceInfo                   *configinfo.ServiceInfo
+	envoyConfigOptions            options.ConfigGeneratorOptions
+	configIdFromServiceConfigFile string
```

Comment:
Where is this variable used? Something seems wrong, this should definitely be set somewhere right?